src/Pages/Login/Login.scss

```
@@ -0,0 +1,159 @@
+@import "../../Styles/common.scss";
+.Login {
+  background-color: #f6f6f6;
+  height: 1000px;
+  h1 {
+    padding-top: 120px;
+    padding-bottom: 20px;
+    font-size: 34px;
+    text-align: center;
+  }
+  main {
+    width: 648px;
+    padding: 32px;
+    margin: 0 auto;
```

padding도 있고 margin도 있는데 하나로 줄일 수 있을 거 같애요!
```suggestion
    margin: 32px auto;
```